api/tests/core/mails/transactional/bookings/booking_confirmation_to_beneficiary_test.py

```diff
@@ -95,7 +96,7 @@ def test_should_return_event_specific_data_for_email_when_offer_is_an_event_send
 
 @freeze_time("2021-10-15 12:48:00")
 def test_should_return_event_specific_data_for_email_when_offer_is_a_duo_event_sendinblue():
-    booking = booking = BookingFactory(
+    booking = BookingFactory(
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏:ok_hand: